src/plonk/mod.rs

```diff
@@ -236,6 +236,24 @@ pub struct RelaxedPlonkTrace<C: CurveAffine> {
     pub W: RelaxedPlonkWitness<C::Scalar>,
 }
 
+#[derive(Debug)]
+pub struct RelaxedPlonkTraceArgs {
```

Comment:
maybe RelaxedPlonkTraceParam?

Reply:
So this is `protogalaxy::Accumulator`.
The module name is part of the structure name

Reply:
It's NBD and if you insist, you can, but we have a lot of domain `params` within the system

We can do it through the `Builder` pattern, then it will have a slightly different API, but there will be no naming issues